build.gradle

```diff
@@ -35,7 +35,6 @@ apply plugin: 'jsonschema2pojo'
 apply plugin: 'jacoco'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
-apply plugin: 'com.github.spotbugs'
 apply plugin: 'com.github.kt3k.coveralls'
```

Comment:
Can you double check that this line can be deleted

Reply:
From what I could find, the `apply plugin` is only required for versions of Gradle older than 2.1 (https://plugins.gradle.org/plugin/com.github.spotbugs). Given we are using v4.9 we should be okay.